sapbtp-operator-charts/templates/deployment.yml

```diff
@@ -112,6 +117,9 @@ spec:
       imagePullSecrets: {{ toYaml .Values.manager.imagePullSecrets | nindent 8 }}
     {{- end }}
       terminationGracePeriodSeconds: 10
+      {{- if .Values.manager.priorityClassName }}
+      priorityClassName: {{ .Values.manager.priorityClassName }}
```

Comment:
kyma project also requires setting priority class for managed applications in kyma-system https://github.com/kyma-project/control-plane/pull/2035